pages/index.jsx

```diff
@@ -1,6 +1,31 @@
+import { useEffect, useState } from 'react';
 import Head from 'next/head';
+import Link from 'next/link';
+
+import apiService from '../services/apiService';
+import LoadingStatus, { loadingStates } from '../components/LoadingStatus';
 
 export default function Home() {
+  const [teamMembers, setTeamMembers] = useState([]);
+  const [teamMemberIds, setTeamMemberIds] = useState([]);
```

Comment:
I think it would be a good idea to not spilt `teamMembers` and `teamMemberIds` into separate variables. The way you use them does work right now but in the future if we need to sort `teamMembers` it will cause some problems. Right now the code works because we currently make the assumption that both `teamMembers` and `teamMemberIds` are arranged in the same order but in the future if we keep these two variables split and we need do some sorting it will require some complex and likely bug prone code to ensure that  both `teamMembers` and `teamMemberIds` are sorted the same way.